### PR TITLE
refactor(`base`): remove preview creation from base

### PR DESCRIPTION
**Pull request by ramfox**
`BaseTabularSchema` is now imported from the `tabular` package in `dataset`

`ConvertBodyFile` has ben renamed to `ConvertFile` and imported from the `dsio` package in `dataset`

Preview creation has been moved to the `preview` package in `dataset`

Refactored references to `CreatePreview` to import it from the `preview` package and adjust to the new function signature. `preview.CreatePreview` expects a fully opened dataset.

### CODE REVIEW

File: remote/browse.go [hunk 1]
```diff
@@ -99,7 +101,48 @@ func (rp LocalPreviews) Preview(ctx context.Context, _, refStr string) (*dataset
...
+			log.Errorf("remote.Preview loading viz script: %w", err.Error())
+			return nil, err
+		}
+		ds.Viz.SetScriptFile(f)
+	}
+
+	if err := ds.OpenBodyFile(ctx, rp.fs); err != nil {
+		log.Errorf("remote.Preview loading body file: %w", err.Error())
+		return nil, err
+	}
```
Comment (ramfox) [inline remote/browse.go:143]: am I crazy? Do we not already have a function that does this? aka loads the dataset & ensures that all the script files are loaded as well?

Comment (b5) [reply remote/browse.go:143]: yar.  `base.OpenDataset`

Comment (ramfox) [reply remote/browse.go:143]: ahhh ty, I was looking in `dsfs`

**Review by b5 — APPROVED** [review]
LGTM

**Pull request merged**